src/Commands/core/ArchiveRestoreCommands.php

```diff
@@ -102,6 +107,9 @@ public function restore(
         array $options = [
             'destination-path' => null,
             'overwrite' => false,
+            'docroot-dir' => self::DOCROOT_DIR,
```

Comment:
We shouldn't need `docroot-dir`. Drupal Finder can determine where the docroot is by examining the composer-installer paths in the composer.json file. Maybe the existing code is already using Drupal Finder?

Reply:
@greg-1-anderson Agree, will replace with DrupalFinder-based implementation